internal/driver/cli.go

```diff
@@ -312,7 +312,8 @@ var usageMsgSrc = "\n\n" +
 	"    -buildid              Override build id for main binary\n" +
 	"    -add_comment          Free-form annotation to add to the profile\n" +
 	"                          Displayed on some reports or with pprof -comments\n" +
-	"    -base source          Source of profile to use as baseline\n" +
+	"    -diff_base source     Source for base profile for comparison\n" +
```

Comment:
Nit here and line below: maybe leave the the "Source of" wording rather than "Source for"? That reads somewhat better to me.

Reply:
Done. Also switched to "of" in parseFlags() function.